cli/command/trust/info.go

```diff
@@ -101,6 +101,38 @@ func lookupTrustInfo(cli command.Cli, remote string) error {
 	}
 	printSignatures(cli, remote, signatureRows)
 
+	roleWithSigs, err := notaryRepo.ListRoles()
+	if err != nil {
+		fmt.Fprintf(cli.Out(), "Error listing signature metadata for roles: %s\n", err)
+		return err
+	}
+
+	signerRoleToKeyIDs := make(map[string][]string)
```

Comment:
can this aggregation be a helper function?  It will make testing simpler because we'll be able to inject mock roles directly